aten/src/ATen/cuda/CUDAGraph.h

```diff
@@ -73,19 +79,16 @@ struct TORCH_CUDA_CPP_API CUDAGraph {
   // Stream on which capture began
   at::cuda::CUDAStream capture_stream_;
 
-  // Default generator on device where capture began
-  at::CUDAGeneratorImpl* capture_gen_;
+  // multiple generator states and their wholegraph_increments in this graph
+  // that are managed by the CUDA Graph
+  std::unordered_map<c10::intrusive_ptr<at::CUDAGeneratorState>, uint64_t>
```

Comment:
interesting...

Reply:
Because this is accessed in the fastpath, I wonder if we should use `flat_hash_map` here. Maybe it's fine.